### PR TITLE
[feature/449901: Build Manage Registration Submissions page view]

### PR DESCRIPTION
**Pull request by shehzad-ismail8**

### CODE REVIEW

**Review by JulianYoungEviden — APPROVED** [review]
All good, no critical items at all.

Going forwards for us, I would like the view models not marked with code-coverage exclusion; we need to find a way to test the usage and presence of required translations in the views - we should be able to prove there are no missing ones.  
And to be honest, a way of proving that the views render the required content...

**Comment by johnfitzgeraldEviden** [comment]
> All good, no critical items at all.
> 
> Going forwards for us, I would like the view models not marked with code-coverage exclusion; we need to find a way to test the usage and presence of required translations in the views - we should be able to prove there are no missing ones. And to be honest, a way of proving that the views render the required content...

View Models should be tested as they could contain logic to help the views

File: src/EPR.RegulatorService.Frontend.UnitTests/Web/Controllers/RegistrationSubmissionsControllerTests.cs [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+            // Assert
+            result.Should().NotBeNull();
+            result.Should().BeOfType<ViewResult>();
+
+            var viewResult = result as ViewResult;
+            viewResult.Should().NotBeNull();
+            Assert.IsInstanceOfType(expectedViewModel, viewResult.Model.GetType());
+
+            var actualBackLink = _controller.ViewBag.CustomBackLinkToDisplay;
```
Comment (johnfitzgeraldEviden) [inline src/EPR.RegulatorService.Frontend.UnitTests/Web/Controllers/RegistrationSubmissionsControllerTests.cs:35]: The Back Link should be part of the view model not passed by ViewBag

File: src/EPR.RegulatorService.Frontend.Web/Controllers/RegistrationSubmissions/RegistrationSubmissionsController.cs [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+                PowerBiLogin = _externalUrlsOptions.PowerBiLogin
+            };
+
+            return View(model);
+        }
+
+        private void SetCustomBackLink()
+        {
+            string pathBase = _pathBase.TrimStart('/').TrimEnd('/');
+            ViewBag.CustomBackLinkToDisplay = $"/{pathBase}/{PagePath.Home}";
```
Comment (johnfitzgeraldEviden) [inline src/EPR.RegulatorService.Frontend.Web/Controllers/RegistrationSubmissions/RegistrationSubmissionsController.cs:43]: See previous comments on Back Link

**Review by johnfitzgeraldEviden — APPROVED** [review]

**Pull request merged**